wiki/Home.md

```diff
@@ -0,0 +1 @@
+Welcome to the wiki!
```

Comment:
I added this dummy page so that the wiki folder is always present in the repo. Otherwise the folder can't be committed.